QuEST/include/QuEST.h

```diff
@@ -1662,6 +1662,43 @@ int measureWithStats(Qureg qureg, int measureQubit, qreal *outcomeProb);
  */
 Complex calcInnerProduct(Qureg bra, Qureg ket);
 
+/** If both input states \p a and \p b are equal-size density matrices
+ * then computes the Hilbert-Schmidt scalar product between \p a and \p b, 
+ * which is equialent to the Frobenius inner product of matrices.
+ * That is, we define the Hilbert-Schmidt scalar product
+ * \f[
+    (( a, b ))_F = ((a, b))_HS := \text{Tr}[ a^\dagger b ] 
```

Comment:
Order of `(( a, b ))_F = ((a, b))_HS` seem backward, but I'd just remove `(( a, b ))_F`; it provides no more info than is in the previous sentence.

Reply:
Addressed this in acf7f95214544204939c8267381218a72ec98096.